File: main/src/main/java/com/sedmelluq/discord/lavaplayer/player/event/PlayerResumeEvent.java
package com.sedmelluq.discord.lavaplayer.player.event;

import com.sedmelluq.discord.lavaplayer.player.AudioPlayer;

/**
 * Event that is fired when a player is resumed.
 */
public class PlayerResumeEvent extends AudioEvent {
  /**
   * @param player Audio player
   */
  public PlayerResumeEvent(AudioPlayer player) {
    super(player);
  }
}

File: main/src/main/java/com/sedmelluq/discord/lavaplayer/manager/event/PlayerResumeEvent.kt
package com.sedmelluq.discord.lavaplayer.manager.event

import com.sedmelluq.discord.lavaplayer.manager.AudioPlayer

/**
 * Event that is fired when a player is resumed.
 * @param player Audio player
 */
data class PlayerResumeEvent(override val player: AudioPlayer) : AudioEvent
